lib/Logger.php

```diff
@@ -36,6 +38,12 @@ public function __construct(array $handlers = array(), Info\InfoInterface $logIn
         }
     }
 
+    public function setAutoRemove($autoRemove = false) {
```

Comment:
I would change
```public function setAutoRemove($autoRemove = false) {```
into
```
public function setAutoRemove($autoRemove)
{
```

A default value for a setter does not make much sense, since it's being explicitly called.

I would also add a getter for the autoRemove property.